content/SCALE/SCALEUIReference/Virtualization/VirtualizationScreens.md

```diff
@@ -127,255 +130,12 @@ The **GPU** screen settings specify graphic processing unit (GPU) for the VM. It
 {{< /expand >}}
 ### Confirm Options Screen
 The **Confirm Options** screen displays the settings selected using the **Create Virtual Machine** wizard screens. It displays the number CPUs, cores, threads, the memory, name of the VM and the disk size.
-
```

Comment:
Put this UI content back in this article. If you want to create a procedure or tutorial then do that but do not move UI content into the tutorial articles. Applies to all the content below this comment.

Reply:
Content restored.